### DISCUSSION TITLE
Add go to hash support to default scrollBehavior

### DISCUSSION THREAD

**Discussion by 13uraksahin** [quasarframework/quasar#17380] [Ideas / Proposals]
### What happened?

Go to hash/id does not work on Quasar created Vue projects.

### What did you expect to happen?

Should work out of the box.

### Reproduction URL

https://stackblitz.com/edit/quasarframework-2fwwrg

### How to reproduce?

go to src/router/index and find: 
`scrollBehavior: (to) => ({ left: 0, top: 0 })`

change with this:
`scrollBehavior: (to) => ({ left: 0, top: 0, el: to.hash || undefined })`

to make it work out of the box, quasar can create project with this addition or with better solution.

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Project Creation (create-quasar), SPA Mode

**Comment by yusufkandemir** [comment]
> Not actually a quasar bug but easy to fix

That's what feature requests for in the discussion section.

Edit: migrated the issue to discussions
